lib/master.js

```diff
@@ -36,6 +36,29 @@ class Master extends BaseService {
         this._firstWorkerStarted = false;
         this._firstWorkerStartupAttempts = 0;
         this.workerStatusMap = {};
+        this.shutdownMasterHandler = () => {
+            this.stop()
+            .then(() => {
+                this._logger.log('info/service-runner/master', 'Exiting master');
+                return this._exitProcess(0);
+            });
+        };
+
+        this.rollingRestartHandler = () => {
+            this._firstWorkerStarted = false;
+            this._firstWorkerStartupAttempts = 0;
+            return this._updateConfig()
+            .then(() => {
+                // Recreate loggers
+                this._logger.close();
+                this._logger = new Logger(this.config.logging);
+            })
+            .then(this._rollingRestart.bind(this));
+        };
+
+        this.startupWorkerExitHandlerMap = {};
+        this.workerExitHandlerMap = {};
+        this.workerMessageHandlerMap = {};
```

Comment:
All of these needs to be prepended with `_`

Reply:
If it's a map, it needs to be a Map

Reply:
Prepended with _ and renamed Dict